csharp/ql/src/Stubs/make_stubs_nuget.py

```diff
@@ -0,0 +1,220 @@
+
+# /output contains the output of the stub generation
+outputDirName = "output"
+outputDir = os.path.join(workDir, outputDirName)
+
+# /output/raw contains the bqrs result from the query, the json equivalent
+rawOutputDirName = "raw"
+rawOutputDir = os.path.join(outputDir, rawOutputDirName)
+os.makedirs(rawOutputDir)
+
+# /output/output contains a dotnet project with the generated stubs
+projectNameOut = "output"
+projectDirOut = os.path.join(outputDir, projectNameOut)
+
+# /db contains the extracted QL DB
+dbName = 'db'
+dbDir = os.path.join(workDir, dbName)
+outputName = "stub"
+outputFile = os.path.join(projectDirOut, outputName + '.cs')
+bqrsFile = os.path.join(rawOutputDir, outputName + '.bqrs')
+jsonFile = os.path.join(rawOutputDir, outputName + '.json')
+version = helpers.get_argv(2, "latest")
+
+print("\n* Creating new input project")
+helpers.run_cmd(['dotnet', 'new', 'classlib', "--language", "C#", '--name',
+                 projectNameIn, '--output', projectDirIn])
+helpers.remove_files(projectDirIn, '.cs')
+
+print("\n* Adding reference to package: " + nuget)
+cmd = ['dotnet', 'add', projectDirIn, 'package', nuget]
+if (version != "latest"):
+    cmd.append('--version')
+    cmd.append(version)
+helpers.run_cmd(cmd)
+
+print("\n* Creating DB")
+helpers.run_cmd(['codeql', 'database', 'create', dbDir, '--language=csharp',
+                 '--command', 'dotnet build /t:rebuild ' + projectDirIn])
+
+if not os.path.isdir(dbDir):
+    print("Expected database directory " + dbDir + " not found.")
+    exit(1)
+
+print("\n* Running stubbing CodeQL query")
+helpers.run_cmd(['codeql', 'query', 'run', os.path.join(
+    thisDir, 'AllStubsFromReference.ql'), '--database', dbDir, '--output', bqrsFile])
+
+helpers.run_cmd(['codeql', 'bqrs', 'decode', bqrsFile, '--output',
+                 jsonFile, '--format=json'])
+
+print("\n* Creating new raw output project")
+rawSrcOutputDirName = 'src'
+rawSrcOutputDir = os.path.join(rawOutputDir, rawSrcOutputDirName)
+helpers.run_cmd(['dotnet', 'new', 'classlib', "--language", "C#",
+                '--name', rawSrcOutputDirName, '--output', rawSrcOutputDir])
+helpers.remove_files(rawSrcOutputDir, '.cs')
+
+# load json from query result file and split it into separate .cs files
+pathInfos = {}
+with open(jsonFile) as json_data:
+    data = json.load(json_data)
+    for row in data['#select']['tuples']:
+        pathInfos[row[3]] = os.path.join(rawSrcOutputDir, row[1] + '.cs')
+        with open(pathInfos[row[3]], 'a') as f:
+            f.write(row[4])
+
+print("\n --> Generated stub files: " + rawSrcOutputDir)
+
+print("\n* Formatting files")
+helpers.run_cmd(['dotnet', 'format', rawSrcOutputDir])
+
+print("\n --> Generated (formatted) stub files: " + rawSrcOutputDir)
+
+print("\n* Processing project.assets.json to generate folder structure")
+stubsDirName = 'stubs'
+stubsDir = os.path.join(outputDir, stubsDirName)
+os.makedirs(stubsDir)
+
+frameworksDirName = 'frameworks'
```

Comment:
`_frameworks` would be better to keep it above all other stubs in the `stubs` folder.

Reply:
@hvitved I pushed an extra commit with this change.